R/seq.R

```diff
@@ -18,10 +18,16 @@ full_seq <- function(x, period, tol = 1e-6) {
 #' @export
 full_seq.numeric <- function(x, period, tol = 1e-6) {
   rng <- range(x, na.rm = TRUE)
-  if (any((x - rng[1]) %% period > tol)) {
+  if (any(((x - rng[1]) %% period > tol) &
```

Comment:
Is it possible to express this more simply by using `abs()`?

Reply:
If we assume period > 0, then %% returns a value between [0, period), and we want to filter on the values at the edges... I suppose you could do something like abs(period/2 - (x - rng[1]) %% period)) <= (period/2 - tol). I think that's marginally faster in terms of vectorized operations, but hard to say if that's "more simply expressed"...

Reply:
Yeah, sounds like the current code is best